### DISCUSSION TITLE
GUI lag on remote server

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#1208] [Help]
The remote GUI displayed by AG are always lagging a bit and slow to react.
Using VNC or Anydesk on the same gigabit network, the GUI on the remote server is fast smoother to use.
Is there a way to have more reactive remote GUI ?

**Comment by apohl79** [comment]
Yeah... I know. VNC has a better algo. The way AG captures the screen is via Apples avfoundation API. And on macOS it can only capture full screen images right now, which get cropped to the UI dimensions and get compressed from there. So the smaller your screen resolution the faster it gets even if your plugin UI stays the same size. I try to avoid retina resolutions on the server side.
